arch/risc-v/src/mpfs/mpfs_start.c

```diff
@@ -65,6 +65,47 @@ volatile bool g_serial_ok = false;
 
 extern void mpfs_cpu_boot(uint32_t);
```

Comment:
```suggestion
```
Seems to be unused

Reply:
sure, though not related to this PR, I didn't add that

Reply:
thanks!